### PR TITLE
AB#39086 AB#39087 AB#39088 Added get team tabs, get team apps, and get team RSCs sample queries

### PR DESCRIPTION
**Pull request by jerryxihe**
ADO Board Links: [Task #39086](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_workitems/edit/39086), [Task #39087](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_workitems/edit/39087), and [Task #39088](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_workitems/edit/39088)

Copied user mode Teams sample queries, but with new generated GUIDs and added information in the tip to ensure the user knows to switch to user mode in order to get Teams IDs and Channel IDs.

### CODE REVIEW

**Review by lhao03 — APPROVED** [review]
nice! (～￣▽￣)～

**Review by gracet9182 — APPROVED** [review]
(●'◡'●) noice

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -2983,6 +2983,35 @@
     }
   ],
   "TeamsAppSampleQueries": [
-      
+    {
+        "id": "a748f894-bb43-4585-8a34-cc093d91c946",
+        "category": "Microsoft Teams",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
```

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -2983,6 +2983,35 @@
...
+        "method": "GET",
+        "humanName": "tabs in a channel",
+        "requestUrl": "/beta/teams/{team-id}/channels/{channel-id}/tabs?$expand=teamsApp",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/channel-list-tabs?view=graph-rest-beta&tabs=http",
+        "tip": "This query requires a team id and a channel id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams. To find the channel id of channels you belong to, stay in user (delegated permissions) mode and run: GET https://graph.microsoft.com/beta/teams/{team-id}/channels.",
+        "skipTest": false
+    },
+    {
+        "id": "428bde00-a605-42cc-90b3-e2b50c359c70",
+        "category": "Microsoft Teams",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
```

File: sample-queries/sample-queries.json [hunk 3]
```diff
@@ -2983,6 +2983,35 @@
...
+        "method": "GET",
+        "humanName": "apps in a team",
+        "requestUrl": "/beta/teams/{team-id}/installedApps?$expand=teamsAppDefinition",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/team-list-installedapps?view=graph-rest-beta&tabs=http",
+        "tip": "This query requires a team id. To find the team id of teams you belong to, you can switch to user (delegated permissions) mode and run: GET https://graph.microsoft.com/v1.0/me/joinedTeams.",
+        "skipTest": false
+    },
+    {
+        "id": "4c676bb1-c267-4a96-ac05-91d3db93b8ee",
+        "category": "Microsoft Teams",
```
Comment (ayubun) [inline sample-queries/sample-queries.json]:
```suggestion
        "category": "Microsoft Teams (beta)",
```

**Review by ayubun — CHANGES_REQUESTED** [review]
These should have the beta tags, since they are beta queries 👀

File: sample-queries/sample-queries.json (hunk 1 above)
Comment (jerryxihe) [reply sample-queries/sample-queries.json]: 1f67d39225be29228094213a74aff3e6505dbcaa

File: sample-queries/sample-queries.json (hunk 2 above)
Comment (jerryxihe) [reply sample-queries/sample-queries.json]: 1f67d39225be29228094213a74aff3e6505dbcaa

File: sample-queries/sample-queries.json (hunk 3 above)
Comment (jerryxihe) [reply sample-queries/sample-queries.json]: 1f67d39225be29228094213a74aff3e6505dbcaa

**Review by ayubun — APPROVED** [review]

**Pull request merged**